src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\NugetPackagesCommon.props" />
   <PropertyGroup Label="Configuration">
-    <Version>103.0.0</Version>
-    <PackageReleaseNotes>Remove pointless Unit.Return method; add a few overloads for ToUnitReturningFunc</PackageReleaseNotes>
+    <Version>103.0.5</Version>
```

```suggestion
    <Version>104.0.0</Version>
```

(technically breaking change albeit trivial)

...maar whatever, zo ook prima.